stimuli_ci_generation/src/server/pubsub.py

```diff
@@ -44,23 +44,21 @@ def callback(future):
     return callback
 
 
-def pub_msg(msg: str, topic_id: str, participant_id: str, experiment_id: str):
+def pub_msg(msg: str, topic_id: str, identifier: str):
     """
     Publish a message to a pubsub topic
     Args:
         msg: message to be published
         topic_id: pubsub topic id
-        participant_id: participant id
-        experiment_id: experiment_id
+        identifier: survey identifier
     """
 
     data = msg.encode("utf-8")
     topic_path = publisher.topic_path(PROJECT_ID, topic_id)
     future = publisher.publish(
         topic_path,
         data,
-        participant_id=participant_id,
-        experiment_id=experiment_id,
+        identifier=identifier,
```

Comment:
so either way works here. It's just a matter of where we have the logic to parse out `participant_id` and `experiment_id`. Not important though since we're deprecating this pubsub code